cli/src/pcluster/models/imagebuilder.py

```diff
@@ -528,7 +527,7 @@ def delete(self, force=False):  # noqa: C901
             try:
                 if AWSApi.instance().cfn.stack_exists(self.image_id):
                     if self.stack.imagebuilder_image_is_building:
-                        raise ImageBuilderActionError(
+                        raise LimitExceededImageBuilderActionError(
```

Comment:
This I believe translates into a 429 (Too Many Requests) response. Probably a 400 (Bad Request) is more appropriate here?